src/main/java/com/google/cloud/spanner/hibernate/SpannerTableExporter.java

```diff
@@ -41,17 +45,36 @@ public SpannerTableExporter(SpannerDialect spannerDialect) {
   }
 
   @Override
-  public String[] getSqlCreateStrings(Table exportable, Metadata metadata) {
-    return new String[]{"test_placeholder"};
+  public String[] getSqlCreateStrings(Table table, Metadata metadata) {
+    /* The current implementation does not support interleaved tables for collections
```

Comment:
Is there an issue to track this?

Reply:
Yes, #17 mentioned in the PR description is about this.